cmake/functions.cmake

```diff
@@ -95,6 +133,11 @@ function(ml_add_library _target _type)
 
   add_compile_definitions(BUILDING_lib${_target})
 
+  if (WIN32 AND _type STREQUAL "SHARED")
+    ml_generate_resources(lib${_target}.dll)
+    list(APPEND PLATFORM_SRCS ${CMAKE_CURRENT_BINARY_DIR}/lib${_target}.rc)
```

Comment:
Just to clarify, is this implying that we no longer need to call the resource compiler directly because Visual Studio will do it automatically if we just append the resource file to the list of source files?

Reply:
> Just to clarify, is this implying that we no longer need to call the resource compiler directly because Visual Studio will do it automatically if we just append the resource file to the list of source files?

Yes, that is the case. CMake neatly handles the magic of generating the resource file for us.